libraries/AP_ADSB/sagetech-sdk/sgEncodeInstall.c

```diff
@@ -52,8 +52,7 @@
 static void checkInstallInputs(sg_install_t *stl)
 {
     assert(
-        0x000000 <= stl->icao && stl->icao <= 0xFFFFFF &&
-        "ICAO address is out of bounds");
+        stl->icao <= 0xFFFFFF && "ICAO address is out of bounds");
```

Comment:
Is icao=0 allowed?

Reply:
From what my coworker told me, ICAO of 0 is permitted to be encoded and sent in our message. Our software will throw up a warning but they didn't want to preclude sending of an ICAO of 0 in this encode function.